packages/augment-api/package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "@polkadot/api": "^10.4.0",
     "@polkadot/typegen": "^10.4.0",
+    "@types/node": "^20.4.9",
```

Comment:
node 16 is still our default so that's probably safer?

Reply:
Solved.